systemtests/src/test/java/io/enmasse/systemtest/bases/TestBase.java

```diff
@@ -81,15 +82,19 @@ protected static void deleteAddressSpace(AddressSpace addressSpace) throws Excep
         }
     }
 
+    protected static void deleteAllAddressSpaces() throws Exception {
+            TestUtils.deleteAllAddressSpaces(addressApiClient, logCollector);
+    }
+
     protected AddressSpace getSharedAddressSpace() {
         return null;
     }
 
     @BeforeEach
     public void setup() throws MalformedURLException {
-        if (addressApiClient == null) {
```

Comment:
Why you delete this? Now it will create new AddressApiClient before each test.

Reply:
ah damn, definitely deleted by accident. restored already